playbooks/installation/index.adoc

```diff
@@ -830,7 +830,7 @@ cat /etc/yum.repos.d/ ...
 ==== Subscribing directly to Red Hat
 
 ----
-ansible -i c1-ocp.myorg.com/hosts OSEv3 -a 'subscription-manager register --username bob@acme.com --password='mypassword'
+ansible -i c1-ocp.myorg.com/hosts OSEv3 -a 'subscription-manager register --username bob@acme.com --password='mypassword''
```

Comment:
Looks like those quotes need to be escaped.

Reply:
Should we just remove the unnecessary quotes around the password parameter?  That would simplify the command.  @etsauer

Reply:
@themoosman as long as the command works, I think thats the better way